centos-ci/bootstrap.sh

```diff
@@ -51,5 +51,5 @@ export DISPLAY=:19
 
 
 echo "$(print_date): running integration tests"
-time scl enable rh-python35 sclo-vagrant1 -- sh -xc "cd ../.. && pipenv shell 'cd integration-tests && behave --no-color --junit; exit \\\$?'"
+time scl enable rh-python35 sclo-vagrant1 -- sh -xc 'cd ../../integration-tests && behave --no-color --junit; exit \\\$?'
```

Comment:
That's what I thought too, but when I had that, the version with a forced divide-by-zero error still reported `exit 0`